files/zh-cn/web/javascript/reference/global_objects/error/cause/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Error: cause"
+slug: Web/JavaScript/Reference/Global_Objects/Error/cause
+---
+
+{{JSRef}}
+
+{{jsxref("Error")}} 实例中的 **`cause`** 数据属性指示导致该错误的具体原始原因。
+
+在捕获错误时，我们可能会使用更具体或更加实用的信息对错误进行包装，再将其重新抛出。`cause` 属性就用于这一场景，以便仍然可以访问原始的错误。
+
+## 取值
+
+它通过 `options.cause` 参数被传入 [`Error()`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Error) 构造函数，并且有可能不存在。
+
+{{js_property_attributes(1, 0, 1)}}
+
+## 描述
+
+`cause` 的值可以是任何类型。你不应该假设捕获错误的 `cause` 属性是一个 `Error`，正如在 `catch` 语句中绑定的变量未必一定是 `Error`。下文 “将结构化的数据作为 `cause`” 示例展示了刻意不提供 `Error` 作为 `cause` 的情况。
+
+## 示例
+
+### 重新抛出包含 `cause` 的错误
+
+有时，捕获错误并使用新的错误信息将其重新抛出是很有用的。在这种情况下，你应该将原始错误传递给新的 `Error` 的构造函数，如下所示：
+
+```js
+try {
+  connectToDatabase();
+} catch (err) {
+  throw new Error('Connecting to database failed.', { cause: err });
+}
+```
+
+更详细的示例请参阅 [错误 > 区分相似的错误](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Error#differentiate_between_similar_errors)。
```

Comment:
https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E7%AB%99%E5%86%85%E9%93%BE%E6%8E%A5%E7%BF%BB%E8%AF%91

```suggestion
更详细的示例请参阅[错误 > 区分相似的错误](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Error#区分相似的错误)。
```